task_messages_protocol/task.proto

```diff
@@ -7,13 +7,27 @@ message TaskResult {
   ResultType result_type = 3;
   optional string output_esdl = 4;
   string logs = 5;
+  repeated TaskEsdlMessage esdl_messages = 7;
 
   enum ResultType {
     SUCCEEDED = 0;
     ERROR = 1;
   }
 }
 
+message TaskEsdlMessage {
```

Comment:
`Task` needed? (also looking at https://github.com/Project-OMOTES/omotes-sdk-protocol/pull/33/files)

or maybe **Task**EsdlMessage and **Job**EsdlMessage

Reply:
duplication removed